stories/core/Header.stories.tsx

```diff
@@ -0,0 +1,209 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Meta, Story } from '@storybook/react/types-6-0';
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+
+import {
+  DropdownMenu,
+  getUserColor,
+  Header,
+  HeaderProps,
+  IconButton,
+  Input,
+  MenuItem,
+  UserIcon,
+} from '../../src/core';
+import HeaderButton from '../../src/core/Header/HeaderButton';
+import HeaderLogo from '../../src/core/Header/HeaderLogo';
+import HeaderBreadcrumbs from '../../src/core/Header/HeaderBreadcrumbs';
+import SvgHelpCircularHollow from '@itwin/itwinui-icons-react/cjs/icons/HelpCircularHollow';
+import SvgVersion from '@itwin/itwinui-icons-react/cjs/icons/Pin';
+import SvgNetwork from '@itwin/itwinui-icons-react/cjs/icons/Network';
+import SvgNotification from '@itwin/itwinui-icons-react/cjs/icons/Notification';
+
+export default {
+  title: 'Core/Header',
+  component: Header,
+  subcomponents: {
+    HeaderBreadcrumbs,
+    HeaderLogo,
+    HeaderButton,
+  },
+  argTypes: {
+    appLogo: { control: { disable: true } },
+    breadcrumbs: { control: { disable: true } },
+    actions: { control: { disable: true } },
+    userIcon: { control: { disable: true } },
+    menuItems: { control: { disable: true } },
+    children: { control: { disable: true } },
+  },
+} as Meta<HeaderProps>;
+
+const buildClickHandler = (menu: string, close: () => void) => (
+  index: number,
+) => {
+  action(`Menu '${menu}', Item #${index} clicked!`)();
+  close();
+};
+
+const buildMenu = (menu: string) => (close: () => void) => [
+  <MenuItem key={1} value={1} onClick={buildClickHandler(menu, close)}>
+    {menu} item #1
+  </MenuItem>,
+  <MenuItem key={2} value={2} onClick={buildClickHandler(menu, close)}>
+    {menu} item #2
+  </MenuItem>,
+  <MenuItem key={3} value={3} onClick={buildClickHandler(menu, close)}>
+    {menu} item #3
+  </MenuItem>,
+];
+
+export const full: Story<HeaderProps> = (args) => {
```

Comment:
Use PascalCase. I don't remember if we had some troubles with camelCase.

Reply:
Nice, I based my example on the only that was wrong, found it odd too... (ErrorPage.stories)